package.json

```diff
@@ -25,6 +27,7 @@
     "@typescript-eslint/parser": "^5.59.9",
     "eslint": "^8.42.0",
     "husky": "^8.0.3",
+    "prettier": "3.0.3",
```

Comment:
Missing lock file. Important, If you update dependencies always push/check the lock file.